C7/UIElements/RightClickMenu.cs

```diff
@@ -0,0 +1,174 @@
+using Godot;
+using C7GameData;
+using C7Engine;
+
+public class RightClickMenu : VBoxContainer
+{
+	protected Game game;
+
+	protected RightClickMenu(Game game) : base()
+	{
+		this.game = game;
+
+		// Set theme for menu node. TODO: This should be made moddable. I noticed in the Godot docs something about loading themes from files
+		// but didn't look into how it works, but that's probably what we'll want to do.
```

Comment:
Those would be the .tres files... we have a C7 theme and a main menu theme so far.  I didn't know you could load them programmatically though, I'd only set them per-scene.  The Godot editor is pretty decent for seeing how they look as you work on them.

But I agree, that can be done at some point in the future.  Feel free to write up an issue to research that so we remember it some day.